larq_compute_engine/core/bgemm_kernels_ruy.h

```diff
@@ -91,12 +91,15 @@ struct BgemmKernel<ruy::Path::kStandardCpp, LhsScalar, RhsScalar, DstScalar,
           TBitpacked rhs_val = Element(rhs, k, j);
           accum += xorpopcount<TBitpacked, AccumScalar>(lhs_val, rhs_val);
         }
+        // Backtransform can still be done in int32
+        accum = spec.backtransform_add - 2 * accum;
```

Comment:
Should we do something like this here, or am I missing an edge case?
```suggestion
        accum = spec.backtransform_add - (accum << 1);
```

Reply:
Wouldn't the compiler do this automatically?